generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -5,12 +5,16 @@
     kill_vm_on_error = yes
     login_timeout = 240
     reboot_count = 25
-    slots_mem = 4
-    maxmem_mem = 32G
     setup_hugepages = yes
-    mem_devs = hugepage
-    size_mem_hugepage = 256M
-    backend_mem = memory-backend-file
```

Comment:
@CongLi  I think it is ok to delete the previous scenario.  From @huangyum,  pc-dimm is not must for hugepage test,  and fixed slot and maxmem is inappropriate, current scenario is match with polarion case.